Src/Campus.Master.API/UI/src/app/modules/calendar/calendar.component.sass

```diff
@@ -1,11 +1,80 @@
+@import "src/styles/styles"
```

Comment:
Consider replacement to import alias @import "styles"

Reply:
Done